ci/cloudbuild/builds/lib/cmake.sh

```diff
@@ -23,6 +23,7 @@ if ((CI_CLOUDBUILD_BUILDS_LIB_CMAKE_SH__++ != 0)); then
 fi # include guard
 
 source module ci/cloudbuild/builds/lib/features.sh
+source module ci/cloudbuild/builds/lib/ctest.sh
```

Comment:
It looks like there are just 12 files that expect to get `ctest::common_args` from cmake.sh.  Should we just update them instead of adding this?

Reply:
I am doing that in #12807, let's merge that and then I will rebase and complete this PR.